doc/reference.md

```diff
@@ -29,6 +29,7 @@ Method                                                       | Return Object
 [completeCardEntry](#completecardentry)                      | void                      | Closes the card entry form on success.
 [showCardNonceProcessingError](#showcardnonceprocessingerror)| void                      | Shows an error in the card entry form without closing the form.
 [setIOSCardEntryTheme](#setioscardentrytheme)                | void                      | Sets the customization theme for the card entry view controller in the native layer.
+[startBuyerVerificationFlow](#startbuyerverificationflow)    | void                      | Starts buyer verfication flow for card-on-file (cof). Displays verification view for some geographies.
```

Comment:
We should reword this. Again, it's not technically for only card on file, but any payment source id. You should reword it as:

Starts the buyer verification for a given payment source id. The most likely use case will be to pass in a card-on-file (cof). This will display a verification view to the user for some geographies to address Strong Customer Authentication.